src/tests/system/tests/test_access_control_ldap_filter.py

```diff
@@ -0,0 +1,251 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control access_filter
+
+LDAP Filters are not intenteded to work with IPA provider
+(there are other mechanisms in IPA to handle it way better than ldap filter),
+thus we are skipping IPA provider
+"""
+
+from __future__ import annotations
+
+import pytest
+
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+from sssd_test_framework.roles.ad import AD
+from sssd_test_framework.roles.samba import Samba
+from sssd_test_framework.roles.ldap import LDAP
+from sssd_test_framework.roles.ipa import IPA
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+@pytest.mark.require(lambda provider: not isinstance(provider, IPA), "It is not testable with IPA")
```

Comment:
I think we should change the message to, 
``` Not supported by IPA.``` or even ```Not supported.```

Technically, it's testable, but it will just fail. =D